src/com/owncloud/android/ui/adapter/ExpandableUploadListAdapter.java

```diff
@@ -93,7 +93,11 @@ public String getGroupName() {
         public Comparator<OCUpload> comparator = new Comparator<OCUpload>() {
             @Override
             public int compare(OCUpload lhs, OCUpload rhs) {
-                return compareUploadId(lhs, rhs);
```

Comment:
It is the name used by default in http://developer.android.com/intl/es/reference/java/util/Comparator.html.
Anyway I've changed it.